mitodl/open-discussions
# Add readonly moderator and contributor tabs

#### What are the relevant tickets?
Part of #864 

#### What's this PR do?
Adds a members tab with moderator and contributor tabs inside it. These list the current moderators and contributors respectively. There is no UI to edit or delete moderators and contributors yet, that will be next PR.

#### How should this be manually tested?
As a moderator, click the edit channel link. Click on the members tab then each of the two tabs inside it. You should see a list of moderators and contributors.

Note that you will see a blank row for the system user which created the channel. At some point we should remove it but it's out of scope of this issue: #891
